doc/reference/storages.rst

```diff
@@ -135,7 +135,7 @@ Demo storages are configured using the ``demostorage`` section::
 ``demostorage`` sections can contain up to 2 storage subsections,
 named ``base`` and ``changes``, specifying the demo storage's base and
 changes storages.  See :meth:`ZODB.DemoStorage.DemoStorage.__init__`
-for more on the base anc changes storages.
+for more on the base and changes storages.
```

Comment:
See ZODB.DemoStorage.DemoStorage.__init__
for more on the base and changes storages.